README.md

```diff
@@ -196,76 +196,33 @@ import com.rudderstack.generated.*
+There are two ways to get started with Rudder-Typer in your browser:
 
-#### Using the `require()`method
+#### 1. Using CDN (Snippet)
 
-```javascript
-// Import RudderStack JS SDK and initialize it
-const RudderAnalytics = require('@rudderstack/analytics-js');
-
-const rudderAnalytics = new RudderAnalytics();
-rudderAnalytics.load(WRITE_KEY, DATA_PLANE_URL, {});
+- Paste the snippet from Rudderstack dashboard to your HTML file. [Ref](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-javascript-sdk/quickstart/#using-cdn)
+- If you are using typescript add `@rudderstack/analytics-js` as dev dependency.
 
-// Import your auto-generated RudderTyper client:
-const rudderTyper = require('./analytics/index');
+```sh
+npm install --save-dev  @rudderstack/analytics-js
+```
 
-// Pass in your @rudderstack/analytics-js instance to RudderTyper client
-rudderTyper.setRudderTyperOptions({
-  analytics: rudderAnalytics,
-});
+- Execute the following command to generate a bundle from the RudderTyper client:
 
-// Issue your first RudderTyper track call!
-rudderTyper.orderCompleted({
-  orderID: 'ck-f306fe0e-cc21-445a-9caa-08245a9aa52c',
-  total: 39.99,
-});
+```sh
+npx browserify analytics/index.js --standalone rudderTyper >  rudderTyperBundle.js
 ```
 
-#### Using `browserify`
-
-- Execute the following command to generate a bundle from the RudderTyper client:
+- For typescript project add `tsify` npm package as dev dependency and run the following command:
 
 ```sh
-browserify analytics/index.js --standalone rudderTyper >  rudderTyperBundle.js
+npx browserify analytics/index.js -p [ tsify ] --standalone rudderTyper >  rudderTyperBundle.js
 ```
 
-- Now you can make calls from your `html` file as shown:
+- Now import your RudderTyper client and send events
 
 ```html
-<head>
-  <script>
-    rudderanalytics = window.rudderanalytics = [];
-    var methods = [
-      'load',
-      'page',
-      'track',
-      'identify',
-      'alias',
-      'group',
-      'ready',
-      'reset',
-      'getAnonymousId',
-      'setAnonymousId',
-    ];
-    for (var i = 0; i < methods.length; i++) {
-      var method = methods[i];
-      rudderanalytics[method] = (function (methodName) {
-        return function () {
-          rudderanalytics.push([methodName].concat(Array.prototype.slice.call(arguments)));
-        };
-      })(method);
-    }
-    rudderanalytics.load(YOUR_WRITE_KEY, DATA_PLANE_URL);
-    rudderanalytics.page();
-  </script>
-  <script src="https://cdn.rudderlabs.com/v1/rudder-analytics.min.js"></script>
-  <script src="./rudderTyperBundle.js"></script>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Document</title>
-</head>
+<script src="./rudderTyperBundle.js"></script>
```

Comment:
What should be the order of the installation of the SDK snippet and this bundle?

Can you clarify this with an example without using the actual snippet?